emcc.py

```diff
@@ -513,6 +513,16 @@ def filter_emscripten_options(argv):
 
   # ---------------- Utilities ---------------
 
+  # Returns the subresource location for run-time access
```

Comment:
How about moving this to `tools/shared.py`, perhaps a static method on the `JS` class?